src/seismometer/html/iframe.py

```diff
@@ -0,0 +1,7 @@
+from pathlib import Path
+
+from IPython.display import IFrame
+
+
+def load_as_iframe(path: Path, height: int = 300) -> IFrame:
```

Comment:
While this is good for caching, and makes sure that any javascript for the new html segment gets run, it does have two drawbacks:

1. VSCode does not support loading an IFrame source from disk - https://github.com/microsoft/vscode/issues/154722
2. JHub requires some handling for this to propagate to jupyter lab instances - https://stackoverflow.com/questions/60257156/not-able-to-embed-jupyterlab-running-on-jupyterhub-into-an-iframe

Reply:
Written up as https://github.com/epic-open-source/seismometer/issues/53